benchmarks/template.bench.ts

```diff
@@ -0,0 +1,107 @@
+// This benchmark is for performance comparing querySelector vs TreeWalker 
+// In the browser, the result may be different
+
+import { Suite } from 'benchmark';
+import 'global-jsdom/register';
+
+export function templateBenchamrk(): void {
+    const root = getRootElement();
+    const suite = new Suite();
+    suite
+    .add('Tree Walker', () => {
+        treeWalker(root);
+    })
+    .add('Query Selector', () => {
+        querySelector(root);
+    })
+    .on('cycle', function(event: any) {
```

Comment:
Codacy has a fix for the issue: [Unexpected any. Specify a different type.](https://app.codacy.com/gh/lroskoshin/horo/pullRequest?prid=7370969)
```suggestion
    .on('cycle', function(event: unknown) {
```